### ISSUE TITLE
Mansion suggestion and issue(?)

### ISSUE THREAD

**Issue by Matthysse** [Lemonszz/Biome-Makeover#87]
Hello!
I have a suggestion regarding the upgraded mansion structure: they should, like the vanilla variant, appear with cobblestone columns under them. I think it's better because with the small islands they create right now, it looks weird on cliffs, and the long pit leading to the basement is visible from the exterior which looks weird: here for example:
![2022-02-23_19 29 51](https://user-images.githubusercontent.com/32510957/155858502-413ee250-4aff-486a-a63b-6beac774f28a.png)
The cobblestone columns would hide this entrance, and make the build look more stable, more menacing I think.

Additionally, it seems there is a spacing issue (at least on forge): I got 2 mansions right next to each other, something that I never saw in vanilla, as they are supposed to be rare:
![2022-02-22_13 09 06](https://user-images.githubusercontent.com/32510957/155858563-d0672500-062a-48f7-bbbe-19c4954fdbcb.png)

Thanks again for your amazing mod!

**Comment by Lemonszz** [comment]
Vanilla did that cobblestone fill method because at the time mansions were added, they didn't have the ability to modify the terrain to  fit around the structure.
We don't plan on using that method, but some fancy pillars might be a cool addition, but it's a pretty low priority as mansions generating like that is fairly rare and not game breaking.

We decreased the rarity of the mansions, as the rewards are honestly not that great. looks like you got lucky in seeing them that close.
